simple-chat-front/src/utils/apiRequests.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+export default function apiRequests() {
+  return axios
+    .get("https://api.github.com/users/AxcelEspinal")
```

Comment:
In practice, URLs like this (or for example, the backend once is ready) have to be set up in environment variables rather than plain strings, practice doing that with this URL (here's an [article](https://create-react-app.dev/docs/adding-custom-environment-variables/))